src/server/queries/diffs.js

```diff
@@ -50,15 +40,9 @@ var diffs = module.exports = promise.promisifyAll({
   },
   getAll: function (signatureHash, callback) {
     // return an array of diff objects that have a matching users_dashboards_signature_hash
-    pool.getConnection(function (err, connection) {
-      if (err) {
-        throw new Error(err);
-      }
-      var selectStr = "SELECT FROM diffs WHERE users_dashboards_signature_hash='" + signatureHash + "';";
-      connection.query(selectStr, function (err, results) {
-        callback(err, results);
-        connection.release();
-      });
+    var selectStr = "SELECT FROM diffs WHERE users_dashboards_signature_hash='" + signatureHash + "';";
```

Comment:
Heads up that this needs to be SELECT * FROM diffs, not just SELECT FROM. I have a fix for this in my own PR, but you may want to put the fix in yours too so that this endpoint will work in your PR (can test with postman until we update the commit hook script).